src/main/deploy/pathplanner/paths/AMP L_AN1 Q_CN1.path

```diff
@@ -0,0 +1,80 @@
+{
+  "version": 1.0,
+  "waypoints": [
+    {
+      "anchor": {
+<<<<<<<< HEAD:src/main/deploy/pathplanner/paths/AMP L_AN1 Q_CN1.path
```

Comment:
These markers look like a bad merge.  You may need to clean this up before checking in.

Reply:
Similar across multiple files in this PR.